src/services/syncFeedPerUser.js

```diff
@@ -152,19 +152,19 @@ const syncMainFeedF2 = async (userId) => {
   console.log('END sync main_feed_f2');
 };
 
-const syncMainFeedBroad = async (userId, source = '') => {
-  console.log('START sync main_feed_broad');
-  const relatedUserIds = await findRelatedUserIds(userId);
-  const unrelatedUserIds = await findUnrelatedUserIds(relatedUserIds);
-  const unrelatedAnonUserIds = await findUnrelatedAnonUserIds(relatedUserIds);
-  console.log('***** Start Process Feed Board *****');
-  if (source !== 'registerUser') {
-    await unFollowMainFeedBroad(userId, relatedUserIds);
-  }
-  await followMainFeedBroad(userId, unrelatedUserIds);
-  await followMainFeedBroad(userId, unrelatedAnonUserIds, true);
-  console.log('END sync main_feed_broad');
-};
+// const syncMainFeedBroad = async (userId, source = '') => {
+//   console.log('START sync main_feed_broad');
+//   const relatedUserIds = await findRelatedUserIds(userId);
+//   const unrelatedUserIds = await findUnrelatedUserIds(relatedUserIds);
+//   const unrelatedAnonUserIds = await findUnrelatedAnonUserIds(relatedUserIds);
+//   console.log('***** Start Process Feed Board *****');
+//   if (source !== 'registerUser') {
+//     await unFollowMainFeedBroad(userId, relatedUserIds);
+//   }
+//   await followMainFeedBroad(userId, unrelatedUserIds);
+//   await followMainFeedBroad(userId, unrelatedAnonUserIds, true);
+//   console.log('END sync main_feed_broad');
+// };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `syncMainFeedBroad` has been commented out. If this function is no longer needed, it would be better to remove it entirely rather than leaving it as commented-out code. Commented-out code can lead to confusion and clutter in the codebase.

```diff
- // const syncMainFeedBroad = async (userId, source = '') => {
- //   console.log('START sync main_feed_broad');
- //   const relatedUserIds = await findRelatedUserIds(userId);
- //   const unrelatedUserIds = await findUnrelatedUserIds(relatedUserIds);
- //   const unrelatedAnonUserIds = await findUnrelatedAnonUserIds(relatedUserIds);
- //   console.log('***** Start Process Feed Board *****');
- //   if (source !== 'registerUser') {
- //     await unFollowMainFeedBroad(userId, relatedUserIds);
- //   }
- //   await followMainFeedBroad(userId, unrelatedUserIds);
- //   await followMainFeedBroad(userId, unrelatedAnonUserIds, true);
- //   console.log('END sync main_feed_broad');
- // };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->